hadoop-common-project/hadoop-common/src/main/java/org/apache/hadoop/fs/store/audit/HttpReferrerAuditHeader.java

```diff
@@ -189,7 +203,14 @@ public String buildHttpReferrer() {
     } catch (URISyntaxException e) {
       WARN_OF_URL_CREATION.warn("Failed to build URI for auditor: " + e, e);
       header = "";
+    } catch (RuntimeException e) {
+      // do not let failure to build the header stop the request being
+      // issued.
+      LOG.warn("Failed to construct referred header {}", e.toString(), e);
+      LOG.debug("Full stack", e);
```

Comment:
Line 209 has already printed the exception stack, so this line of code is redundant.

Reply:
I think we should remove the "e" from the warn log and leave debug for the full stack trace.

Reply:
that's what I had intended to do, clearly failed

Reply:
haha! it happens.